app/assets/javascripts/admins/components/CodeCsvCard.js

```diff
@@ -2,12 +2,17 @@ import React from 'react'
 import request from '../../shared/requests/request'
 import { APIRoutes } from '../../shared/routes'
 
+import CodeCsvModal from './CodeCsvModal'
+
 class CodeCsvCard extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
+      isModalOpen: false,
     }
     this.downloadCodeCsv = this.downloadCodeCsv.bind(this)
+    this.openCodeCsv = this.openCodeCsv.bind(this)
+    this.closeCsvModal = this.closeCsvModal.bind(this)
```

Comment:
Since these two have complementary functionalities I would name them in the same fashion: like `openCsvModal` and `closeCsvModal` for example.